### PR TITLE
Update about page and remove workgroups page

### PR DESCRIPTION
**Pull request by berezovskyi**
The information about the old working groups is no longer relevant, removing it.

I have updated the About page and moved it up in the menu to inform about the Open Project. If you wish, we can move/copy some information over to that page from https://github.com/oslc-op/oslc-admin/blob/master/GOVERNANCE.md or https://github.com/oslc-op/oslc-admin/blob/master/CONTRIBUTING.md but I prefer to keep boring text to a minimum.

@axelreichwein & @ndjc this is the OP link I promised to provide to you: https://open-services.net/about/ (after this PR is merged).

### CODE REVIEW

**Comment by berezovskyi** [comment]
@gabrielfdac I have changed how data is rendered in most cases from a list of paragraphs to a single body field (multiline TOML string) that is passed through Markdown processor by Hugo,

**Comment by ndjc** [comment]
No comments. Is this (https://open-services.net/about/) the page I should link to from the specs by default, in place of the old OASIS TC workgroup page?

**Comment by berezovskyi** [comment]
Why @ndjc? We really don't have a Core TC or Domains TC any more. About page is covering the Open Project itself. If I removed some relevant information, please tell me!

Or do you suggest the PGB reinstate Core & Domains WGs?

**Comment by ndjc** [comment]
No, I was not suggesting that the separate WGs be resurrected. In our previous meeting, we were discussing what ReSpec should do in the block of text in the Status section:

> This document was last revised or approved by the [OASIS OSLC Lifecycle Integration Core (OSLC Core) Open Project](https://www.oasis-open.org/committees/oslc-core/) on the above date. The level of approval is also listed above. Check the “Latest version” location noted above for possible later revisions of this document. Any other numbered Versions and other technical work produced by the Open Project are listed at https://www.oasis-open.org/committees/tc_home.php?wg_abbrev=oslc-core#technical.
> 
> Project members should send comments on this specification to the project's email list. Others should send comments to the project’s public comment list oslc-core-comment@lists.oasis-open.org, after subscribing to it by following the instructions at the “Send A Comment” button on the project’s web page at https://www.oasis-open.org/committees/oslc-core/.

We agreed that the three references to OASIS TC pages should be replaced by references to the project's home page. My question in my previous comment is: Is https://open-services.net/about/ what I should use for those references by default, if no conf variable is set to override those links? I know we are still working on the mailing list replacement.

**Comment by berezovskyi** [comment]
@ndjc yes, open-services.net/about is the right URL.

@gabrielfdac I messed up the navbar anchor links like Mission, Leadership etc. Could you please take a look and help me fix it?

@OSLC/oslc-pgb please check the text.

File: content/about/_index.md [hunk 1]
```diff
@@ -3,36 +3,67 @@ categories = []
...
+heroTitle = "About the OSLC Open Project"
+heroSubtitle = "Since May 2019, OSLC is an OASIS Open Project. Learn more below."
 
 mission_title = "Our Mission"
 mission_subtitle = "What we are trying to achieve"
-mission_content = [
-  "The community is striving to pull together many diverse initiatives focused around application integration solutions to collaborate and partner with each other rather than compete. There are many organizations with different backgrounds sharing the vision of integrated engineering environments. Sometimes they are promoting different approaches than OSLC and linked data solutions, but the goal of the community is to openly discuss complementary approaches which can also be appropriate with the goal of improving the overall viability of standards/implementations for supporting industry integration challenges."
-]
+mission_body = """
+The community is striving to pull together many diverse initiatives focused
```
Comment (jamsden) [inline content/about/_index.md]: This feels out of date. OSLC is an established, mature and well-practiced approach to integration. We don't need to talk about competition, but rather we should focus on what integration OSLC supports and what we are doing in the future in oslc-op. Suggest this needs to be re-written to better indicate where we are and where we're going.

File: content/about/_index.md [hunk 2]
```diff
@@ -3,36 +3,67 @@ categories = []
...
-governing_company = [
-  "IBM",
-  "KTH Royal Institute of Technology",
-  "Siemens",
-  "Koneksys",
-]
+participation_title = "Participating in OSLC"
+participation_subtitle = "Participation in the OSLC Open Project is open anyone who has something to contribute."
+participation_body = """
+The community is striving to pull together many diverse initiatives focused
```
Comment (jamsden) [inline content/about/_index.md]: This is also out of date and doesn't sufficiently address how to participate and contribute. Suggest "Contributing to OSLC" as this is more appropriate for an open project.

File: content/about/_index.md [hunk 3]
```diff
@@ -64,6 +95,23 @@ sponsors_others_list = [
...
 
+history_title = "History of OSLC"
+history_subtitle = "What we are trying to achieve"
+history_body = """
+
+- Early work predating OSLC started as an internal IBM effort in 2009.
+- OSLC as an organisation comprising multiple independent members in the Steering Committee started in 2009 and first specs were published that year.
+- Work on a common Core spec started in 2010.
+- Key people behind OSLC participated in the W3C Linked Data Platform recommendation development between 2012 and 2015.
+- OSLC independent Steering Committee decides to move OSLC governance to OASIS in 2013.
```
Comment (jamsden) [inline content/about/_index.md]: Needs updating to include addition of domains TC, and then migration to oslc-op.

**Comment by berezovskyi** [comment]
Thanks Jim! I spent almost 2 hours on this yesterday, which is far more than I planned. Could you please edit the source files directly and then me and Gabriel make sure it compiles?

--
/Andrew
(from phone)



On Tue, Jun 18, 2019 at 2:15 PM +0200, "Jim Amsden" <notifications@github.com<mailto:notifications@github.com>> wrote:


@jamsden requested changes on this pull request.

Thanks Andrew for doing this. I have requested a few changes.

________________________________

In content/about/_index.md<https://github.com/OSLC/oslc-site-hugo/pull/254#discussion_r294773002>:

>
 mission_title = "Our Mission"
 mission_subtitle = "What we are trying to achieve"
-mission_content = [
-  "The community is striving to pull together many diverse initiatives focused around application integration solutions to collaborate and partner with each other rather than compete. There are many organizations with different backgrounds sharing the vision of integrated engineering environments. Sometimes they are promoting different approaches than OSLC and linked data solutions, but the goal of the community is to openly discuss complementary approaches which can also be appropriate with the goal of improving the overall viability of standards/implementations for supporting industry integration challenges."
-]
+mission_body = """
+The community is striving to pull together many diverse initiatives focused


This feels out of date. OSLC is an established, mature and well-practiced approach to integration. We don't need to talk about competition, but rather we should focus on what integration OSLC supports and what we are doing in the future in oslc-op. Suggest this needs to be re-written to better indicate where we are and where we're going.

________________________________

In content/about/_index.md<https://github.com/OSLC/oslc-site-hugo/pull/254#discussion_r294773211>:

> -governing_members = [
-  "Jim Amsden",
-  "Bill Chown",
-  "Andrew Berezovskyi",
-  "Axel Reichwein",
-]
-governing_company = [
-  "IBM",
-  "KTH Royal Institute of Technology",
-  "Siemens",
-  "Koneksys",
-]
+participation_title = "Participating in OSLC"
+participation_subtitle = "Participation in the OSLC Open Project is open anyone who has something to contribute."
+participation_body = """
+The community is striving to pull together many diverse initiatives focused


This is also out of date and doesn't sufficiently address how to participate and contribute. Suggest "Contributing to OSLC" as this is more appropriate for an open project.

________________________________

In content/about/_index.md<https://github.com/OSLC/oslc-site-hugo/pull/254#discussion_r294775104>:

> @@ -64,6 +95,23 @@ sponsors_others_list = [
   "Software AG"
 ]

+history_title = "History of OSLC"
+history_subtitle = "What we are trying to achieve"
+history_body = """
+
+- Early work predating OSLC started as an internal IBM effort in 2009.
+- OSLC as an organisation comprising multiple independent members in the Steering Committee started in 2009 and first specs were published that year.
+- Work on a common Core spec started in 2010.
+- Key people behind OSLC participated in the W3C Linked Data Platform recommendation development between 2012 and 2015.
+- OSLC independent Steering Committee decides to move OSLC governance to OASIS in 2013.


Needs updating to include addition of domains TC, and then migration to oslc-op.

**Comment by jamsden** [comment]
You can release as is, we'll get to fixing it later.

**Comment by berezovskyi** [comment]
merged to master and deployed

**Comment by berezovskyi** [comment]
but the https://open-services.net/workgroups/ is still up

@gabrielfdac could you please help with that?

**Comment by berezovskyi** [comment]
By the way, we got the first usage of the new page: I saw that https://github.com/EricssonResearch/oslc_prolog points to the old wiki and added two banners there to point to the new OP page: 

- https://wiki.oasis-open.org/oslc-core/FrontPage
- https://wiki.oasis-open.org/oslc-sc/FrontPage

**Pull request merged**